test/unit/utils/primitive.test.js

```diff
@@ -1,32 +1,24 @@
 const path = require('path')
 
-const childProcessMock = require('child_process')
-const fsMock = require('fs')
+const execaMock = require('execa')
```

Comment:
This should be import.

Reply:
Other files are still using require, too. Let's move for `/src/` completely to import syntax.